diem-unified-charter.md

```diff
@@ -1,49 +1,94 @@
+Until the architecture document is published, the working group will develop and maintain an informational use cases and requirements document, but is not required to request publication of this document.
+
+### Investigation of Serialization Formats
+
+The working group will research, analyze and evaluate application layer serialization formats for suitability to the initial scope.
+The working group will attempt to match the conventions for serializing data structures used in the discovery protocols, for example, key value strings in DNS records or structured field values in http.
+This work must be completed in the use cases and requirements document, and the architecture must not assume any specific serialization formats or discovery mechanisms.
+
+### Investigation of Securing Mechanisms
+
+The working group will research, analyze and evaluate securing mechanisms for digital emblems.
+The group is not required to describe object level securing mechanisms, in the case that existing protocol level securing mechanisms are sufficient, for example DNSSEC for DNS records, or TLS for well known HTTPS URLs.
+This work must be completed in the use cases and requirements document, and the architecture must not assume any specific securing mechanism or discovery mechanisms.
+
+### Design of Emblem Types
+
+The working group will describe at least one digital emblem suitable for indicating protection under international humanitarian law.
+The working group will describe at least one digital emblem suitable for indicating the presence of a hazard or a certification of the absence of a hazard.
+This work must be completed in the use cases and requirements document, and must not assume any specific serialization format or discovery mechanisms.
+
+### Use Cases and Requirements
+
+The working group will develop an informational document describing use cases and requirements for the initial proposed standards.
+Although this document is listed as a deliverable, it is not required to be published as an RFC.
+
+### Architecture
+
+The working group will develop an informational document describing the architecture, terminology and data model of digital emblems.
+This document must not assume any specific serialization formats, securing mechanisms, discovery mechanisms, or use cases.
+This document must be published as an RFC, before any proposed standard documents can be adopted by the working group.
+
+### Initial Serialization
+
+A proposed standard describing the first serialization format for digital emblems.
+This document may describe a mandatory to implement securing mechanism.
+If a securing mechanism is described, this document must describe at least one mandatory to implement cryptographic algorithm which is already supported by the securing mechanism.
+This document must be published as a proposed standard RFC, and must not assume any specific discovery mechanisms, or use cases.
```

Comment:
```suggestion
This document must be published as a proposed standard RFC, and must not assume any specific use cases.
```

Reply:
Removing this to enable discovery mechanism and serialization to be more closely aligned